UOP1_Project/Assets/Scripts/Sound/SoundManager.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Collections;
+using System.Linq;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+public class SoundManager : MonoBehaviour
+{
+	public static SoundManager singleton;
+
+	[Header ("Init Variables")]
+	[Tooltip ("Amount of sound emitters created on Start")]
+	public int initialPoolSize = 1;
+	[Tooltip ("Frequency at which the sound manager will delete extra emitters that are not in use")]
+	public float trimFrequency = 5;
+	[Tooltip ("Extra time after instanced emitters are not in use anymore before destroying them")]
+	public float extraTimeBeforeTrim = 5;
+
+	[Header ("Mixer Groups")]
+	public AudioMixerGroup masterMixer;
+	public AudioMixerGroup sfxMixer;
+	public AudioMixerGroup musicMixer;
```

Comment:
Are these used anymore?

Reply:
Not anymore, they were just common groups used for the first implementation. I will delete these variables in the next commit.